theano/gof/tests/test_vm.py

```diff
@@ -279,9 +279,12 @@ def time_linker(name, linker):
                 print pre.ru_ixrss, post.ru_ixrss
                 print pre.ru_idrss, post.ru_idrss
                 print pre.ru_maxrss, post.ru_maxrss
-
+        print 1
```

Comment:
No, those function aren't run by default. We must in another shell check the memory usage with top. So this allow us to know when we exchange between the function being tested.